mne/viz/epochs.py

```diff
@@ -442,6 +438,249 @@ def plot_epochs(epochs, epoch_idx=None, picks=None, scalings=None,
     return fig
 
 
+def plot_epochs(epochs, picks=None, scalings=None, n_epochs=20,
+                n_channels=10, fig_title=None, show=True, block=False):
+    """ Visualize single trials.
+
+    Bad epochs can be marked with a left click on top of the epoch.
+    Calling this function drops all the selected bad epochs as well as bad
+    epochs marked beforehand with rejection parameters.
+    The scaling can be adjusted with 'page up' and 'page down'.
+
+    Parameters
+    ----------
+
+    epochs : instance of Epochs
+        The epochs object
+    picks : array-like of int | None
+        Channels to be included. If None only good data channels are used.
+        Defaults to None
+    scalings : dict | None
+        Scale factors for the traces. If None, defaults to:
+        `dict(mag=1e-12, grad=4e-11, eeg=20e-6, eog=150e-6, ecg=5e-4, emg=1e-3,
+             ref_meg=1e-12, misc=1e-3, stim=1, resp=1, chpi=1e-4)`
+    n_epochs : int
+        The number of epochs per view. Defaults to 20.
+    n_channels : int
+        The number of channels per view. Defaults to 10.
+    fig_title : str | None
+        The title of the window. If None, epochs name will be displayed.
+        Defaults to None.
+    show : bool
+        Show figure if True. Defaults to True
+    block : bool
+        Whether to halt program execution until the figure is closed.
+        Useful for rejecting bad trials on the fly by clicking on an epoch.
+        Defaults to False.
+
+    Returns
+    -------
+    fig : Instance of matplotlib.figure.Figure
+        The figure.
+    """
+    import matplotlib.pyplot as plt
+    import matplotlib as mpl
+    from matplotlib.collections import LineCollection
+    from matplotlib.colors import colorConverter
+    scalings = _handle_default('scalings_plot_raw', scalings)
+    color = _handle_default('color', None)
+    bad_color = (1.0, 0.0, 0.0)
+
+    epoch_data = epochs.get_data()
+    if picks is None:
+        picks = _handle_picks(epochs)
+    if len(picks) < 1:
+        raise RuntimeError('No appropriate channels found. Please'
+                           ' check your picks')
+    n_epochs = np.min([n_epochs, len(epochs.events)])
```

Comment:
you can use min from std lib here rather than creating a list than an array than calling a fortran routine :)